### PR TITLE
[8.0] Restore DatiCassaPrevidenziale handling: when present, add it as invo…

### PR DESCRIPTION
**Pull request by sergiocorato**
…ice line.

Reverting part of https://github.com/OCA/l10n-italy/pull/703

### CODE REVIEW

**Comment by sergiocorato** [comment]
@OCA/local-italy-developers qualche reviewer?

**Comment by sergiocorato** [comment]
@scigghia @OpenCode una review please?

**Comment by sergiocorato** [comment]
@eLBati è tutto regolare quindi procederei con il merge

**Comment by sbidoul** [comment]
@eLBati @sergiocorato you changed this PR after asking the bot to merge.
Be sure the changes you need are on branch 8.0-ocabot-merge-pr-1015-by-sergiocorato-bump-no, if it's not what you want, delete that branch to stop the bot.

**Comment by sergiocorato** [comment]
> @eLBati @sergiocorato you changed this PR after asking the bot to merge.
> Be sure the changes you need are on branch 8.0-ocabot-merge-pr-1015-by-sergiocorato-bump-no, if it's not what you want, delete that branch to stop the bot.

@sbidoul thanks, how to delete that branch? from command line?

**Comment by eLBati** [comment]
Deleted

File: l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py [hunk 1]
```diff
@@ -792,7 +792,7 @@ def invoiceCreate(
         PaymentTermsModel = self.env['fatturapa.payment_term']
         SummaryDatasModel = self.env['faturapa.summary.data']
 
-        company = self.env.user.company_id
+        company = self.env.user.company_id.with_env(self.env)
```
Comment (eLBati) [inline l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py]: ?

**Comment by sergiocorato** [comment]
> Deleted

Thanks, @sbidoul @eLBati I thought the message 'let's try again' was referred to the user, so ocabot has stopped to work on this.

**Comment by sbidoul** [comment]
@sergiocorato @eLBati the message "let's try again" means the bot detected something else was committed on 8.0 after creation of  8.0-ocabot-merge-pr-1015-by-sergiocorato-bump-no, so it rebased again to make sure the PR was tested on top of the latest 8.0 before merging.

File: l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py (hunk 1 above)
Comment (sergiocorato) [reply l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py]: Ne avevamo discusso, riporto qui:
```
self.env.user.company_id.cassa_previdenziale_product_id.with_env(self.env).env.user
res.users(20,)
self.env.user
res.users(20,)
self.env.user.company_id.cassa_previdenziale_product_id.env.user
res.users(1,)
```
l'ultima riga è quella presente adesso, che mi dà admin a priori

anche sulla 10 v. https://github.com/odoo/odoo/issues/9649

Comment (sergiocorato) [reply l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py]: è un problema solo per il multicompany

Comment (eLBati) [reply l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py]: Quindi può essere un problema di tutte le versioni?

**Pull request closed without merging**